scenario/smartsearch.go

```diff
@@ -0,0 +1,462 @@
+package scenario
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"sync"
+	"time"
+	"unicode"
+
+	"github.com/goccy/go-json"
+	"github.com/pkg/errors"
+	"github.com/qlik-oss/enigma-go/v3"
+	"github.com/qlik-oss/gopherciser/action"
+	"github.com/qlik-oss/gopherciser/connection"
+	"github.com/qlik-oss/gopherciser/enummap"
+	"github.com/qlik-oss/gopherciser/helpers"
+	"github.com/qlik-oss/gopherciser/logger"
+	"github.com/qlik-oss/gopherciser/session"
+)
+
+const (
+	// typeOneCharDuration is based on:
+	//
+	// typing speed: 50 words per miniute
```

Comment:
* minute

also general note (no need to change anything) "word" is extremely language specific and not a good metric, it's better to base it on Characters per minute as it would be language independent.

Reply:
fixed